pkg/sql/catalog/systemschema/system.go

```diff
@@ -387,8 +387,9 @@ CREATE TABLE system.protected_ts_records (
    num_spans INT8 NOT NULL, -- num spans is important to know how to decode spans
    spans     BYTES NOT NULL,
    verified  BOOL NOT NULL DEFAULT (false),
+   target    BYTES,
```

Comment:
What is target? Can we put commentary here on on the column in the table descriptor. Raw bytes columns scare me. I recognize spans was that but it was at least a little clearer what was going to go in there.

Reply:
done.